### DISCUSSION TITLE
adding assertNotEmittedTo method

### DISCUSSION THREAD

**Discussion by omaresmael** [livewire/livewire#5047] [3. Feature ideas]
hey guys, I have a feature idea, that I think it might be useful

### Problem

I have a situation where I have multiple components that I emit an event to them based on some conditions
let's say that we have component A and component B 
 I want to test that an event is emitted to component A and NOT emitted to component B 
 I could achieve the first part of the test by the `assertEmittedTo` method. but for now, the second part is not achievable 
 
 ### Proposal 
add the `assertNotEmittedTo` method so I could test if the event is not been emitted to a particular component.

### Approach
my approach is similar to what we have for the `assertEmittedTo` 
- assert that the event was emitted
- assert that the event was Not emitted to the component

I've created this method on my local, I was asking if you would like me to create a pr, or do you have concerns that I'm missing about this idea?

thank you 💟

**Comment by joshhanley** [comment]
@omaresmael yeah makes sense, PR's are welcome! 🙂

**Reply by omaresmael** [reply]
[done](https://github.com/livewire/livewire/pull/5126)
please have a look when you have time 
thank you
